src/scene_map.cpp

```diff
@@ -250,6 +253,7 @@ void Scene_Map::FinishTeleportPlayer() {
 void Scene_Map::CallBattle() {
 	Main_Data::game_data.system.before_battle_music = Game_System::GetCurrentBGM();
 	Game_System::SePlay(Game_System::GetSystemSE(Game_System::SFX_BeginBattle));
+	Game_System::BgmPlay(Game_System::GetSystemBGM(Game_System::BGM_Battle));
```

Comment:
Why is this needed? `Scene_Battle::Start()` will start the music too.

Reply:
Because the battle music already starts when the screen flash appears on the map.
The call in Start() is still needed for the BattleTest.

Reply:
That was a mistake on my part, I forgot to take it away when putting the commit, my bad. I'll update it and take it away.